routes/conversations.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+const Router = require('express').Router;
+const Conversation = require('../models/Conversation');
+const Message = require('../models/Message');
+const sgMail = require('@sendgrid/mail');
+sgMail.setApiKey(process.env.SENDGRID_API_KEY);
+
+const router = Router();
+
+
+router.post('/', (req, res) => {
+
```

Comment:
remove extra newline

Reply:
Im going to add in lint to avoid these small things in the future. Done.